dec.py

```
@@ -0,0 +1,95 @@
+# Method 1: Raw DEFLATE (no header)
+print("\n1. Trying raw DEFLATE decompression...")
+try:
+    decompressed = zlib.decompress(data, -zlib.MAX_WBITS)
+    print("✓ SUCCESS with raw DEFLATE!")
+    print(f"Decompressed size: {len(decompressed)} bytes")
+    print("\nFirst 1000 characters of decompressed data:")
+    print("-" * 60)
+    try:
+        text = decompressed.decode("utf-8")
+        print(text[:1000])
+    except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)